var/spack/repos/builtin/packages/freesasa/package.py

```diff
@@ -30,6 +30,8 @@ class Freesasa(AutotoolsPackage):
     depends_on("json-c", when="+json")
     depends_on("libxml2", when="+xml")
 
+    patch("libcpp.patch", when="@2.1.2")
```

Comment:
Can you a comment above the path explaining:
1. Why the patch is needed
2. What is the effect of the patch
3. Where the patch was taken from

?

Reply:
I did not take the patch from anywhere.